### ISSUE TITLE
Documentation workflow (feature request)

### ISSUE THREAD

**Issue by soundstep** [pullreminders/backlog#195]
Hi,

We started using Pull panda, and it is great so far.

Feature request:
It would have been nice to have any kind of step by step workflow example to understand the process, in your documentation.

One thing we can't figure out is how to change this status of "Waiting on".

Our current workflow:

1. Create pull request
2. Assign a team for review, 2 users get assigned by pull assigner
    Current status in Slack with `/pullreminders ` is `waiting on reviewers`
3. Users review the code and request changes
    Current status in Slack with `/pullreminders ` is `waiting on developer`
4. Developer make the changes
4. Developer mark all conversations as resolved
5. Developer re-request review (with the little circle next the reviewer in the PR), Github review little icon back to orange dot:

![Screen Shot 2019-09-05 at 14 37 33](https://user-images.githubusercontent.com/133293/64346813-beafcd00-cfea-11e9-89b4-9ce2d8453e4a.png)

Expected:
Status changed to `waiting on reviewers`

Current:
Status stays on `waiting on developer`, which shows on the Slack pull reminders.

Is our workflow with Github wrong?
